src/components/EditableField/EditableField.tsx

```diff
@@ -414,7 +427,26 @@ export class EditableField extends React.Component<
     }
   }
 
-  renderInputFields() {
+  renderAddButton = () => {
+    const { disabled } = this.props
+    const { fieldValue, multipleValuesEnabled } = this.state
+
+    const isLastValueEmpty = fieldValue[fieldValue.length - 1].value === ''
+    const isSingleAndEmpty = fieldValue.length === 1 && isLastValueEmpty
```

Comment:
I believe `isSingleAndEmpty` is redundant because `isLastValueEmpty` will return `true` if the array has one item and is empty AND it will return true if it has n items and the last one is empty.

Reply:
The reason I have both, is because I use one to decide to render, and the other to decide if it should disabled, I know it looks a little bit weird... 😅

Reply:
Right, I see the minutia there. Thanks!